python/electrostatic_force.py

```diff
@@ -0,0 +1,362 @@
+from tkinter import *
+import compute
+def com1():
+    global lavel
+    charge1val=Charge1.get()
+    #print(charge1val)
+    charge2val=Charge2.get()
+    #print(charge2val)
+    root.destroy()
+    root1=Tk()
+    global flag
+    root1.title("OUTPUT")
+    root1.geometry("7000x6000")
+    root1.minsize(500,600)
+    canvas=Canvas(root1,width=2000,height=2200,bg="sky blue")
```

Comment:
Codacy found an issue: ['Canvas' may be undefined, or defined from star imports: tkinter (F405)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6279653)